catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/TestCswFilterDelegate.java

```diff
@@ -828,7 +818,7 @@ private static FilterCapabilities getMockFilterCapabilities() {
         comparisonOpsList.add(ComparisonOperatorType.GREATER_THAN_EQUAL_TO);
         comparisonOpsList.add(ComparisonOperatorType.LESS_THAN);
         comparisonOpsList.add(ComparisonOperatorType.LESS_THAN_EQUAL_TO);
-        comparisonOpsList.add(ComparisonOperatorType.BETWEEN);
+        comparisonOpsList.add(BETWEEN);
```

Comment:
@vinamartin we imported `ComparisonOperatorType.BETWEEN` because it was used multiple times in this class, whereas the other Types are only used once in this class.